### DISCUSSION TITLE
SQLite 数据库，ServerTime=DateTimeKind.Local后插入的日期值不含毫秒？

### DISCUSSION THREAD

**Discussion by ecocom** [dotnetcore/FreeSql#1143] [General]
```c#
[Column(CanUpdate = false, ServerTime = DateTimeKind.Local)]
public virtual DateTime CreatedTime { get; set; }
```
SQLite 数据库插入的 CreatedTime 毫秒部分为0，当前开发项目需要精确到毫秒。
调式看到SQLite 用的datetime(current_timestamp,'localtime') ，获取到的值不含毫秒。SqlServer 用的 getdate() ，含毫秒，其它库尚未测。能否考虑 各库都确保给加上毫秒值？（当然也可以去掉 ServerTime 直接 DateTime.Now 解决）
谢谢

**Comment by 2881099** [comment]
如果要高精度，建议用 long 时间戳保存

**Reply by ecocom** [reply]
ok, 谢谢
